src/Atmos/Model/moisture.jl

```diff
@@ -107,8 +107,14 @@ function flux_moisture!(moist::EquilMoist, atmos::AtmosModel, flux::Grad, state:
   flux.moisture.ρq_tot += u_tot * state.moisture.ρq_tot
 end
 
-function flux_diffusive!(moist::EquilMoist, flux::Grad, state::Vars, diffusive::Vars, aux::Vars, t::Real, D_t)
+function flux_diffusive!(moist::EquilMoist, atmos::AtmosModel, flux::Grad, state::Vars, diffusive::Vars, aux::Vars, t::Real, D_t)
   d_q_tot = (-D_t) .* diffusive.moisture.∇q_tot
+
+  z  = altitude(atmos.orientation, aux)
+  w_sub = subsidence_velocity(atmos.subsidence, z)
+  k̂ = vertical_unit_vector(atmos.orientation, aux)
+  flux.moisture.ρq_tot += state.ρ*w_sub .* k̂ .* diffusive.∇q_tot
```

Comment:
Same as for energy, it needs to go into source:
```suggestion
  source.moisture.ρq_tot += state.ρ*w_sub .* k̂ .* diffusive.∇q_tot
```